pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BrokerResourceOnlineOfflineStateModelFactory.java

```diff
@@ -97,14 +97,19 @@ public void onBecomeOnlineFromOffline(Message message, NotificationContext conte
 
     @Transition(from = "ONLINE", to = "OFFLINE")
     public void onBecomeOfflineFromOnline(Message message, NotificationContext context) {
-      String tableNameWithType = message.getPartitionName();
-      LOGGER.info("Processing transition from ONLINE to OFFLINE for table: {}", tableNameWithType);
+      String physicalOrLogicalTable = message.getPartitionName();
```

Comment:
Why is this method called for logical table ? Is it because an entry is removed from `brokerResource` ?

Reply:
Yes.